lizard_auth_client/tests.py

```diff
@@ -1071,6 +1089,107 @@ def test_organisation_index_view_superuser(self):
         response = views.ManageOrganisationIndex.as_view()(request)
         self.assertEqual(response.status_code, 200)
 
+    def get_new_user_sso_mock(self, override_user_data=None, status_code=201):
+        user_data = {'username': 'root',
+                     'first_name': 'willie',
+                     'last_name': 'wortel',
+                     'email': 'noreply@example.com',
+                     'is_active': True,
+                     'is_staff': False,
+                     'is_superuser': False}
+
+        if override_user_data:
+            user_data.update(override_user_data)
+
+        return mock.patch(
+            'lizard_auth_client.views.requests.post',
+            return_value=MockResponse(
+                status_code,
+                {'success': True,
+                 'user': user_data}))
+
+    def get_new_user_response(self, post_data):
+        request = self.request_factory.post(
+            reverse('lizard_auth_client.management_users_add',
+                    args=[self.organisation_1.id]))
+        request.session = {}
+        request.user = self.user_1
+        request.user.is_superuser = True
+        request.POST = post_data
+
+        return views.ManageUserAddView.as_view()(
+            request, self.organisation_1.id)
+
+    def test_organisation_add_totally_new_user(self):
+        """A manager should be able to add an user
+           with an unknown email address and username
+        """
+        with self.get_new_user_sso_mock(status_code=201):
+            response = self.get_new_user_response(
+                {'username': 'root_new',
+                 'first_name': 'willie',
+                 'email': 'noreply_new@example.com'})
+
+            # Should return an HTTP redirect to detail page
+            self.assertEqual(response.status_code, 302)
+
+    def test_organisation_add_known_exact_match_user(self):
+        """A manager should be able to add an user
```

Comment:
Gaat dit helemaal goed? Een nieuwe gebruiker aanmaken die al bestaat? Eigenlijk zou je eerst gezocht moeten hebben en 'em moeten hebben gevonden, zou dit geen 'duplicate' http error terug moeten geven?

Aan de andere kant maakt het niet zoveel uit, want er wordt geen schade aangericht.

Reply:
Eh, sorry, dit is de client, niet de server. Oepsie.